pkgs/applications/misc/mediaelch/default.nix

```diff
@@ -23,13 +23,13 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "mediaelch";
-  version = "2.8.18";
+  version = "2.10.0";
 
   src = fetchFromGitHub {
     owner = "Komet";
     repo = "MediaElch";
     rev = "v${version}";
-    sha256 = "sha256-9kwU9j8YNF/OmzrQaRAlBpW+t/tIpZJw5+pfEoTmCBA=";
+    sha256 = "sha256-hipOOG+ibfsJZKLcnB6a5+OOvSs4WUdpEY+RiVKJc+k=";
     fetchSubmodules = true;
```

Comment:
Can we use our quazip?

Reply:
The package already has `-DUSE_EXTERN_QUAZIP=ON`, which [means](https://github.com/Komet/MediaElch/blob/ed356f44e513b92cb1551e03f80e530991328356/CMakeLists.txt#L31) using our quazip.

Reply:
Which also means that `fetchSubmodules = true;` is not needed.